### DISCUSSION TITLE
Simulated Satellite To Satellite Tracking for GNSS

### DISCUSSION THREAD

**Discussion by laplace76** [groops-devs/groops#110] [Ideas]
Dear Developers,

I'd like to know whether the current version of groops is able to simulate satellite-to-satellite observations (pseudorange and Doppler) between pairs of GNSS satellites or not, and if it is possible to add such observations to the GNSS orbit determination process defined by the cookbook's example, in order to assess the effect of SST w.r.t. a reduction of the GNSS receivers number on the ground.

Best regards

**Comment by tmayerguerr** [comment]
Hi,

At the moment it is not possible to simulate satellite-to-satellite observations between GNSS satellites.

We will think about it, but I don't think we will have time for it in the near future. We are only a small group and pretty busy with the usual project work, conferences and teaching. Next on our agenda is Beidou and processing of SLR observations. I'm sorry I can't give a better answer at this time.
 
regards
Torsten
